product_security/red_hat_responsibilities/dedicated_network_security.adoc

```diff
@@ -0,0 +1,98 @@
+[[dedicated-network-security]]
+= Network Security
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+
+[[network-segregation]]
+== Network Segregation
+
+{product-title} uses a software-defined networking (SDN) approach to provide a
+unified cluster network that enables communication between pods across the
+cluster. This pod network is established and maintained by the OpenShift SDN,
+which configures an overlay network using Open vSwitch (OVS).
+
+{product-title} uses the *ovs-networkpolicy* plug-in, which allows project
+administrators to configure their own isolation policies using NetworkPolicy
+objects. These policies can be fully managed by project owners and
+`dedicated-admin` users to add network-level isolation for the projects.
+
+[[network-segregation-iptables]]
+=== Iptables
+
+{product-title} has an iptables-based implementation of the service-routing
+infrastructure. It uses probabilistic iptables rewriting rules to distribute
+incoming service connections between the endpoint pods.
+
+[[network-segregation-security-groups]]
+=== Security Groups
+
```

Comment:
Add following text:
Security Groups provide security at the protocol and port access level. Each security group works similar to a firewall and has a set of rules that filter traffic coming into and out of a node.